src/systemcmds/tests/test_mixer.cpp

```diff
@@ -604,7 +591,7 @@ mixer_callback(uintptr_t handle, uint8_t control_group, uint8_t control_index, f
 
 	if (should_prearm && control_group == actuator_controls_s::GROUP_INDEX_ATTITUDE &&
 	    control_index == actuator_controls_s::INDEX_THROTTLE) {
-		control = NAN_VALUE;
+		control = std::numeric_limits<float>::quiet_NaN();
```

Comment:
My one concern about this line is that `std::numeric_limits<float>::quiet_NaN()` can actually return a number that is not Nan if they are unsupported for the datatype. You can check their support by putting an assert on `std::numeric_limits<float>::has_quiet_NaN` to make sure. Alternatively you could also use the NAN macro here as was done in the mixer.cpp as I believe this macro is only defined if the NAN exists.